src/components/OSCALDrawerSelector.js

```diff
@@ -0,0 +1,67 @@
+import { Divider, Drawer, IconButton, styled, useTheme } from "@mui/material";
+import React from "react";
+import ChevronLeftIcon from "@mui/icons-material/ChevronLeft";
+
+import TreeView from "@mui/lab/TreeView";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import ChevronRightIcon from "@mui/icons-material/ChevronRight";
+import TreeItem from "@mui/lab/TreeItem";
+
+const DrawerHeader = styled("div")(({ theme }) => ({
+  display: "flex",
+  alignItems: "center",
+  padding: theme.spacing(0, 1),
+  // necessary for content to be below app bar
+  ...theme.mixins.toolbar,
+  justifyContent: "flex-end",
+}));
+
+const drawerWidth = 200;
+
+export default function OSCALDrawerSelector(props) {
+  const theme = useTheme();
+
+  return (
+    <Drawer
+      sx={{
+        width: drawerWidth,
+        flexShrink: 0,
+        "& .MuiDrawer-paper": {
+          width: drawerWidth,
+          boxSizing: "border-box",
+        },
+      }}
+      variant="persistent"
+      anchor="left"
+      open={props.open}
+    >
```

Comment:
We want to do this like you do for `DrawerHeader` above. We should generally _heavily_ avoid the `sx` prop except for in limited, short styles.

```ts
const Drawer = styled(BaseDrawer)`
  width: ${drawerWidth};
  flex-shrink: 0;
  & .MuiDrawer-paper {
    width: ${drawerWidth};
    box-sizing: "border-box";
  }
`;
```